src/plots.py

```diff
@@ -151,27 +152,43 @@ def plot_histo(
         if layered
         else {}
     )
-
     st.vega_lite_chart(
         data=df,
         spec={
             **CONFIG_MAIN,
             "mark": {"type": "bar", **CONFIG_MARK, "blend": "normal"},
             "params": config_params,
+            "transform": [
+                {"bin": True, "field": col_x, "as": "bin_col_x", "maxbins": bin},
+                {
+                    "aggregate": [{"op": "count", "as": "xCount"}],
+                    "groupby": ["bin_col_x", "bin_col_x_end"],
+                },
+                {"joinaggregate": [{"op": "sum", "field": "xCount", "as": "TotalCount"}]},
+                {"calculate": "datum.xCount/datum.TotalCount", "as": "PercentOfTotal"},
```

Comment:
I'm not sure why but when you add up the percentage in the histogram (eg for 2 bins), it doesn't add up to 100%.

Reply:
Trying with the titanic dataset and age variable, the sum is not equal to 100% due to the presence of NaNs. I'll add a filter to discard them when normalizing (without ordinal being checked).